config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  namespace :api, defaults: { format: 'json' } do
+    resources :greetings, only: %i[index]
+  end
```

Comment:
- [ ] Let's create a main static route. (should be marked as `root`). The root webpage is the one that displays requests that include only the domain/port part of the URL and not the resource path (e.g., http://localhost:3000/). Kindly add this 😉 👍🏻 

- [ ] And you need to make sure that a static controller has one action !